pallets/messages/src/lib.rs

```diff
@@ -123,6 +124,7 @@ pub mod pallet {
 	/// A temporary storage of messages, given a schema id, for a duration of block period.
 	/// At the start of the next block this storage is cleared and moved into Messages storage.
 	/// - Value: List of Messages
+	/// Obsolete storage (will be removed in next storage migration)
```

Comment:
I think we can remove this here, right - it's empty.

Reply:
Test it on rococo and development.

Reply:
We cannot just test it on frequency-rococo now as that's a testnet and should always be either in line with production, or pre-release for production.

If there isn't a storage migration needed, I'm fine with just removing it.

Reply:
removed after local upgrade test